_test_common.sh

```diff
@@ -55,7 +55,8 @@ function assert_equals {
 		fi
 	done
 
-	if [ -z "${DISPLAY_SUCCESSFUL_TEST_RESULT}" ] && [ "${_TEST_RESULT}" == "true" ]
+	if [["${_TEST_RESULT}" == "true" ]] &&
```

Comment:
@kiwm, missing space after first `[[`.

@kiwm, why did you prefer this syntax instead of this [one](https://github.com/liferay/liferay-docker/blob/e9ae1b280b124412284aef5f722f890841138aee/build_all_images.sh#L634-L635)? Do you have other examples in the existing source code to share using the syntax chosen by you?

Thanks